README.md

```diff
@@ -64,6 +64,21 @@ cp completion/kubie.fish ~/.config/fish/completions/
 
 Then reopen fish or source the file.
 
+
+#### Zsh
+
+Autocomplete with Zsh can either be installed by sourcing the file eg.:
+
+```bash
+source ./completion/kubie.zsh
+```
+
+Or by adding it to your fpath and renaming the completion file to _kubie eg.:
+```bash
+cp .completion/kubie.zsh ~/.zsh/completions
+```
+
```

Comment:
Here you can add extra notes for oh-my-zsh

```bash
cp .completion/kubie.zsh ${ZSH}/custom/completions/_kubie.zsh
```

Reply:
I have done some steps but it's not working for me.

`kubie ns` autocomplete is not working for me

Reply:
`kubie` itself is also not working